docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,640 @@
+    4. `email` (OPTIONAL, string). Email address of the contact person or organization.
+    5. `role` (OPTIONAL, string). Role of the contact person. This field should only be set when the `name` field
+    is set.
+
+### 2. `model_index`
+
+There can be multiple models. For each model the following fields are present.
+
+1. `name` (REQUIRED, string). The name of the model.
+2. `model` (REQUIRED, string). A URI pointing to a repository containing the model file.
+3. `results` (list). There can be multiple results. For each result the following fields are present.
+
+    1. `task` (list).
+
+        1. `task_type` (REQUIRED, string). The task of the model, for example "object-classifcation".
+        2. `task_name` (OPTIONAL, string). A pretty name fo the model taks, for example "Object Classification".
+
+    2. `dataset` (list).
+
+        1. `type` (REQUIRED, string). The type of the dataset, can be a dataset id from [HuggingFace datasets](https://hf.co/datasets)
+        or a link to a repository containing the dataset[^1], for example "common_voice".
+        2. `name` (REQUIRED, string). Name pretty name for the dataset, for example "Common Voice (French)".
+        3. `split` (OPTIONAL, string). The split of the dataset, for example "train".
+        4. `revision` (OPTIONAL, string). Version of the dataset, for example 5503434ddd753f426f4b38109466949a1217c2bb.
+
+    3. `metrics` (list). There can be multiple metrics. For each metric the following fields are present.
+
+        1. `type` (REQUIRED, string). A metric-id from [Hugging Face metrics](https://hf.co//metrics)[^2], for example accuracy.
+        2. `name` (REQUIRED, string). A descriptive name of the metric. For example "false positive rate" is
+        not a descriptive name, but "training false positive rate w.r.t class x" is.
+        3. `value` (REQUIRED, float). The value of the metric.
+        4. `class` (OPTIONAL, string/int/float/bool). Some metrics (such as false positive rate for multiclass classification)
+        depend on a specific output class. In this field the output class can be specified. It must only be set for metrics
+        for which it makes sense.
+        5. `subgroups` (list). Metrics can be computed on subgroups of specific features. For example one can compute the
+        accuracy for examples where the feature "gender" is set to "male".
+        There can be multiple subgroups, which means that the metric is computed on the intersection of those subgroups.
+        For each subgroup the following fields are present.
+
+            1. `feature` (OPTIONAL, string). The name of the feature. For example: "gender".
+            2. `group` (OPTIONAL, string). The value of the feature. If `feature` is set, this field must be set as wel.
+            For example: "male".
+
+    4. `measurements`.
+
+        1. `bar_plots` (list). The purpose of this field is to capute bar plot like measurements, for example SHAP values.
```

Comment:
I think these sections are pretty interesting :) can't wait to see it more into action, as i think for sure measurements are way better then putting in binary/images and in this way we could also more easily compare between different models

Reply:
Yes, I am also very curious to see if this actually captures information at the right granularity!